src/Footer/Footer.js

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react';
+import './Footer.css';
+
+class Footer extends Component {
```

Comment:
Consider making this a stateless functional component: https://javascriptplayground.com/functional-stateless-components-react/